keras/backend/tensorflow_backend.py

```diff
@@ -20,6 +20,9 @@
 from .common import set_image_dim_ordering
 from .common import image_dim_ordering
 
+import logging
```

Comment:
actually, that is a great question since that was the first thing I looked into before start asking for our own instance of it. `theano` also exposes its logging utility so we can plug in to whichever is being used as the backend. Having said that, working with our own logging instance is also nice since people can, e.g. turn off one library's log messages but still hold on to another's and it will also give Keras developers more freedom to define what each level means since they might not have the same utility across groups. 

I ended up leaning more towards having our instance but I have no experience in using it in such a high-performance library. So if people have thoughts/concerns about that, then we can piggy-bag on other logging tools.